.github/workflows/eclint.yml

```diff
@@ -0,0 +1,13 @@
+name: Test Whitespace and line endings
+
+on: [pull_request]
+
+jobs:
+  eclint:
+    name: Editor Config check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: snow-actions/eclint@v1.0.1
```

Comment:
Sorry if I didn't make it clear in the last comment, this also looks the case we can't directly control the eclint version in case we need? (like upstream changes default behavier or something like that breaks the build)

Reply:
ECLint wasn't pinned to a version before. This is pinned to a specific version of the Action itself

Reply:
No, it wasn't pinned before, but we have the ability to pin it any time, by using the action, we lost the control of it.